### PR TITLE
Update the License Url for packages

### PR DESCRIPTION
**Pull request by tarekgh**

### CODE REVIEW

**Comment by tarekgh** [comment]
@weshaggard this similar change we did for coreclr and core-setup. I couldn't find better place to add the temporary target than dir.targets

**Comment by tarekgh** [comment]
@zhenlan PTAL

**Comment by tarekgh** [comment]
https://github.com/dotnet/corefx/issues/12190

**Comment by StephenBonikowsky** [comment]
LGTM

File: dir.targets [hunk 1]
```diff
@@ -107,6 +107,13 @@
     </ItemGroup>
   </Target>
   
+  <!-- OverrideLicenseUrl is temporary till we update the buildtools to v2 -->
```
Comment (weshaggard) [inline dir.targets]: Can you file a tracking issue to clean up these workarounds in the future?

Comment (tarekgh) [inline dir.targets]: I'll do that

**Comment by tarekgh** [comment]
@StephenBonikowsky looks I don't have write access to merge to this repo. could you please help merging it?

File: Packaging.props [hunk 2]
```diff
@@ -22,6 +22,7 @@
     <!-- by default all packages will use the same version which revs with respect to product version -->
     <PackageVersion Condition="'$(PackageVersion)' == ''">4.4.0</PackageVersion>
     <SkipValidatePackageTargetFramework>true</SkipValidatePackageTargetFramework>
+    <LicenseUrl>https://github.com/dotnet/corefx/blob/master/LICENSE</LicenseUrl>
```
Comment (zhenlan) [inline Packaging.props]: Should we use https://github.com/dotnet/wcf/blob/master/LICENSE?

Comment (tarekgh) [inline Packaging.props]: we have a discussion regarding this and we are waiting our PM's to advise. if they advised with changing it I'll go and change this in all repos we did the change in.

Comment (zhenlan) [inline Packaging.props]: Sounds good.

**Comment by tarekgh** [comment]
Thanks @zhenlan could you please merge it? I don't have rights to do it :-(

**Pull request merged**